app/views/layouts/_header.html.erb

```diff
@@ -16,6 +15,7 @@
               <% end %>
                 <li><%= link_to t(".profile_header"), current_user %></li>
                 <li><%= link_to t(".setting_header"), edit_user_path(current_user) %></li>
+                <li><%= link_to t("posts.header"), new_post_path %></li>
```

Comment:
I think it is better to put tab for user can direct create post rather put it into the dropdown bar, if it is a route for user to create a new post.